scheduler/src/cook/compute_cluster.clj

```diff
@@ -419,7 +419,7 @@
                             {:template template})))
         full-cluster-config (-> (:config cluster-definition-template) (merge config) (assoc :dynamic-cluster-config? true))
         cluster (resolved full-cluster-config {:exit-code-syncer-state @exit-code-syncer-state-atom})]
-    (initialize-cluster cluster {:pool-name->fenzo @pool-name->fenzo-atom})))
```

Comment:
How was this ever working?

Reply:
always failed to remove from fenzo when jobs completed. but jobs ran normally otherwise (this is when using dynamic clusters only)